wpilibc/athena/include/RobotBase.h

```diff
@@ -26,8 +26,8 @@ class DriverStation;
     }                                                                        \
     HAL_Report(HALUsageReporting::kResourceType_Language,                    \
                HALUsageReporting::kLanguage_CPlusPlus);                      \
-    static _ClassName_ robot;                                                \
     std::printf("\n********** Robot program starting **********\n");         \
+    static _ClassName_ robot;                                                \
```

Comment:
That printout occurs before some operations that used to occur in StartCompetition() via RobotInit(). They were moved into robot class constructors. To maintain when the print occurs in relation to operations with things like NetworkTables, it was moved to before the robot was constructed. Would be rather have it print after the robot class initializes, regardless of what it does then?